### PR TITLE
Add DOI to Circos now that it is published

### PR DESCRIPTION
**Pull request by hexylena**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/circos/macros.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <macros>
   <token name="@CIRCOS_VERSION@">0.69.8</token>
 
-  <token name="@WRAPPER_VERSION@">@CIRCOS_VERSION@+galaxy6</token>
+  <token name="@WRAPPER_VERSION@">@CIRCOS_VERSION@+galaxy7</token>
```
Comment (bgruening) [inline tools/circos/macros.xml:5]: do we need to bump?
Congrats @hexylena and @shiltemann!

Comment (hexylena) [reply tools/circos/macros.xml:5]: Probably not, but, the "same version" detection and weirdness around installables, I always bump just in case?

Comment (hexylena) [reply tools/circos/macros.xml:5]: Happy to remove, if you prefer.

Comment (bgruening) [reply tools/circos/macros.xml:5]: It's ok... but it's falling :(

Comment (hexylena) [reply tools/circos/macros.xml:5]: Fixed :)

**Pull request merged**